code.cpp

```diff
@@ -0,0 +1,136 @@
+#include <array>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <unistd.h>
+#include <vector>
+
+const int length = 32;
```

Comment:
Use more common 'width' name

Reply:
Also try to avoid global variables, to may be you should create class or structure for table